/home/skyr/miniconda3/envs/ai_stuff/lib/python3.11/site-packages/torch/nn/modules/conv.py:456: UserWarning: Applied workaround for CuDNN issue, install nvrtc.so (Triggered internally at ../aten/src/ATen/native/cudnn/Conv_v8.cpp:80.)
  return F.conv2d(input, weight, bias, self.stride,
Traceback (most recent call last):
  File "/home/skyr/PycharmProjects/MuAlphaZeroBuild/mu_alpha_zero/trainer.py", line 142, in train
    wins_p1, wins_p2, game_draws = self.mcts.self_play(self.network, self.device, self_play_games,
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/skyr/PycharmProjects/MuAlphaZeroBuild/mu_alpha_zero/MuZero/MZ_MCTS/mz_search_tree.py", line 150, in self_play
    game_results = self.play_one_game(net, device, calculate_avg_num_children=game == num_games - 1)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/skyr/PycharmProjects/MuAlphaZeroBuild/mu_alpha_zero/MuZero/MZ_MCTS/mz_search_tree.py", line 56, in play_one_game
    pi, (v, latent) = self.search(network_wrapper, state, player, device, calculate_avg_num_children=(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/skyr/PycharmProjects/MuAlphaZeroBuild/mu_alpha_zero/MuZero/MZ_MCTS/mz_search_tree.py", line 93, in search
    pi, v = network_wrapper.prediction_forward(state_.unsqueeze(0), predict=True)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/skyr/PycharmProjects/MuAlphaZeroBuild/mu_alpha_zero/MuZero/Network/networks.py", line 82, in prediction_forward
    pi, v = self.prediction_network.predict(x, muzero=True)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/skyr/miniconda3/envs/ai_stuff/lib/python3.11/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/skyr/PycharmProjects/MuAlphaZeroBuild/mu_alpha_zero/AlphaZero/Network/nnet.py", line 196, in predict
    pi, v = self.forward(x, muzero=muzero)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/skyr/PycharmProjects/MuAlphaZeroBuild/mu_alpha_zero/AlphaZero/Network/nnet.py", line 189, in forward
    output = th.sum(val_h_output * support_range, dim=1)
                    ~~~~~~~~~~~~~^~~~~~~~~~~~~~~
RuntimeError: The size of tensor a (300) must match the size of tensor b (601) at non-singleton dimension 1